src/sql-parser/src/parser.rs

```diff
@@ -1047,6 +1057,7 @@ impl<'a> Parser<'a> {
                 Token::Keyword(IN) => Precedence::Like,
                 Token::Keyword(BETWEEN) => Precedence::Like,
                 Token::Keyword(LIKE) => Precedence::Like,
+                Token::Keyword(AT) => Precedence::Like,
```

Comment:
I'm not sure if this is the correct precedence.

Reply:
For now this is fine, @benesch wants to clean up our precedence code, which might make it in before this, otherwise we'll clean it up.

For posterity, generally we try to match [the postgres precedence table](https://github.com/postgres/postgres/blob/ff769831e081a4e69147019128a849ced43ec802/src/backend/parser/gram.y#L765-L776) which has a bunch of implicit rules, and a custom one for `AT` that is fairly high precedence.